src/main/java/core/basesyntax/service/impl/CsvFileWriterServiceImpl.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.CsvFileWriterService;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+public class CsvFileWriterServiceImpl implements CsvFileWriterService {
+    @Override
+    public void writeToFile(String data, String filePath) throws IOException {
```

Comment:
The method `writeToFile` should not throw the general `IOException`. It would be better to handle the exception inside the method and rethrow it as a RuntimeException with an informative message if necessary.

Reply:
Implement this comment.